src/constants.py

```diff
@@ -24,7 +24,7 @@
 
 # Integration constants
 PEER_RELATION_NAME = "kratos-peers"
-INTERNAL_INGRESS_RELATION_NAME = "internal-ingress"
+PUBLIC_INGRESS_INTEGRATION_NAME = "public-ingress"
```

Comment:
As far as we removed the internal ingress altogether, does it make sense to name the remaining one just "ingress", and not "public_ingress"?

Reply:
+1

Reply:
For now I think keep it because there is another ingress relation called `admin-ingress` which typically is not often used at the current moment. And also we might add the internal ingress back if needed.